Unit-01/06-sql-alchemy-1/functions.py

```diff
@@ -0,0 +1,4 @@
+def find_snack(right_id, lst):
```

Comment:
since you're using a database, it looks like you don't need this function anymore!